ui/app/common/obs/views/showObservation.html

```diff
@@ -1,25 +1,40 @@
 <li ng-class="::{'is-text': observation.type == 'Text', 'has-child' : observation.groupMembers.length > 0 ,'video-concept':observation.isVideoConcept(), 'image-preview': observation.isImageConcept()}" >
     <div ng-class="::{'is-abnormal': (observation.abnormal ==  true || observation.interpretation === 'ABNORMAL')}" class="tree-list-item clearfix" bm-gallery patient="patient">
-        <span ng-hide="observation.isImageConcept()" ng-if="::(!observation.isImageConcept() || (observation.isImageConcept() && !print))" class="testUnderPanel left">
+        <span ng-show="observation.groupMembers.length!==0" ng-if="::(!observation.isImageConcept() || (observation.isImageConcept() && !print))" class="testUnderPanel left">
+            <label class="obs-label" ng-if="::configIsObservationForImages">{{::(observation.concept.shortName || observation.concept.name)}}</label>
+            <label ng-if="::!configIsObservationForImages">{{::(observation.concept.shortName || observation.concept.name)}}</label>
+            <hint concept-details="::{
+            lowNormal:observation.lowNormal||observation.concept.lowNormal,
+            hiNormal:observation.hiNormal||observation.concept.hiNormal,
+            }"/>
+            <a href="{{'/document_images/' + observation.value}}" download ng-if="observation.isVideoConcept()" class="button btn fr btn-download"><i class="fa fa-download" aria-hidden="true"></i></a>
```

Comment:
`/document_images/` is hardcoded. Isn't this a configurable value, and if someone chooses a diff config, won't this code break?

Reply:
In bahmni-india-package's openmrs container's volume 'bahmni-document-images:/home/bahmni/document_images' is mapped to patient-documents container's 'bahmni-document-images' volume.  Which is similar to bahmni installation's '/home/bahmni/document_images'

Lines from /usr/local/apache2/conf/bahmni-proxy.conf from proxy container:
#Patient Documents
ProxyPass /document_images http://patient-documents:80/document_images
ProxyPassReverse /document_images http://patient-documents:80/document_images

So I feel it is ok to map from /document_images
Please check. Thank you.

Reply:
@N0-man - What do you think? Is this something that might change deployment-to-deployment? For instance, when S3 is used, or when an on-prem installation is chosen? If so, then we will need to make the `/document_images/` path a configuration in UI code.

Reply:
![AllPatientDocument](https://user-images.githubusercontent.com/38578773/188655719-cc4e0162-0643-475a-83b5-1304e7455493.png)
<img width="425" alt="patientDocument" src="https://user-images.githubusercontent.com/38578773/188655729-67dab73d-69f7-47e2-89ea-0c90a925a6af.png">

Old files will follow: patientid-encounterName-uuid.ext (eg. 6-Patient-Document-706a448b-3f10-11e4-adec-0800271c1b75.png)
New ones will follow: patientid_encounterName_uuid__filename.ext (eg. 6-Patient-Document-706a448b-3f10-11e4-adec-0800271c1b75__doc1.png)

Reply:
We want to keep the /document_images intact inorder to protect the documents behind authentication and authorization.

@gsluthra we have a container running nginx + JS `patient-documents` that is protecting the files behind `document_images` - see [this](https://github.com/BahmniIndiaDistro/helm-umbrella-chart/blob/c6c01e9421da1f5559c51d9d0897d18f8a8d6d97/templates/ingress.yaml#L56) and [this](https://github.com/Bahmni/patient-documents/blob/0acfbd837bb3598a00ce6707579ead6487f576d0/package/docker/default.conf.template#L20)